### PR TITLE
HW09 - Initial commit with Husky pre-commit hook

### PR DESCRIPTION
**Pull request by alekseigolov1**

### CODE REVIEW

**Comment by Marusia2** [comment]
Great job in general! Having thorough testing coverage of the API endpoints guarantees that the anticipated behavior remains consistent. I suppose you can add just the final Axios test to the checklist :)

**Comment by alekseigolov1** [comment]
> Great job in general! Having thorough testing coverage of the API endpoints guarantees that the anticipated behavior remains consistent. I suppose you can add just the final Axios test to the checklist :)

Hey, Axios test is at the end of the file , you must have been missed it . thank you for your review, appreciate it

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+| Scenario # | Description                                                                                 |
+| ---------- | ------------------------------------------------------------------------------------------- |
+| 1          | PUT: Correct ID and valid key updates order and returns correct code                        |
+| 2          | PUT : Invalid api-key is rejected as unauthorised                                           |
+| 3          | PUT : Correct return code is returned in case orderID is out of range (BAD_REQUEST          |
+| 4          | DELETE: Correct ID and valid key deletes order and returns success code 204(NO_CONTENT)     |
+| 5          | DELETE: Request without valid key is rejected with correct return code                      |
+| 6          | DELETE: Invalid order ID fails as bad request with correct return code                      |
+| 7          | GET: Correct error code is returned if username and password are correctly set with api-key |
```
Comment (vladimir-ovodenko-inbank) [inline readme.md:9]: looks like positive case for GET is missing

**Comment by Marusia2** [comment]
> > Great job in general! Having thorough testing coverage of the API endpoints guarantees that the anticipated behavior remains consistent. I suppose you can add just the final Axios test to the checklist :)
> 
> Hey, Axios test is at the end of the file , you must have been missed it . thank you for your review, appreciate it

Sorry, I guess I missed them :) Great job anyway, find a lot of interesting solutions :)

**Review by Marusia2 — APPROVED** [review]
Well done :)

**Pull request merged**